wizeline/wize-q-remix
# feat: Added logic to assign a question to a person

#### What does this PR do?
This feature includes a dropdown to select people by department

#### Where should the reviewer start?

app/components/QuestionForm/QuestionForm.Styled.jsx
app/components/QuestionForm/QuestionForm.jsx
app/routes/questions/new.jsx
app/utils/backend/validators/question.js
app/components/DropdownMenu/
app/controllers/employees/
app/routes/employees/

#### How should this be manually tested?
Login
Click on the Ask question button
Select a department 
Verify that the people dropdown is displayed
If there is people displayed, select a person displayed
Verify that you can create the question 


#### Screenshots
<img width="835" alt="image" src="https://user-images.githubusercontent.com/97201781/208967068-64c88edb-cc41-44ed-afb4-aa9559a40077.png">
<img width="841" alt="image" src="https://user-images.githubusercontent.com/97201781/208967148-696724f2-8283-40fd-96e5-6a39cd686e4e.png">
